src/content/nft/index.md

```diff
@@ -0,0 +1,219 @@
+
+_If Andy Warhol had been born in the late 90s, he probably would have minted Campbell's Soup as an NFT. It's only a matter of time before Kanye puts a run of Yeezys on Ethereum. And one day owning your car might be managed with NFTs._
+
+<InfoBanner isWarning>NOTE: we don't currently explain the problems NFTs solve. On defi we talk about the problems with the tradfi system and on daos we talk about tthe difficulties of trusting strangers. The internet today isn't set up for ownership, but how can we easily explain this?</InfoBanner>
+
+## What's an NFT? {#what-are-nfts}
+
+NFTs are tokens that we can use to represent a record of ownership. They let us tokenise unique items like art, collectibles, even real estate. They can only have one owner at a time and they're protected by the Ethereum blockchain – no one can modify the record of ownership or copy/paste a new NFT into existence.
+
+NFT stands for non-fungible token. Non-fungible is an economic term that you could use to describe things like your furniture, a song file, or your computer. These things are not exchangeable for other items because they have unique properties.
+
+Fungibile items on the other hand can be exchanged as their value defines them rather than their unique properties. For example ETH is fungible because 1 ETH is exchangeable for another ETH.
+
+### NFT examples {#nft-examples}
+
+The NFT world is relatively new. In theory, the scope for NFTs is anything that is unique that needs provable ownership. Here are some examples of NFTs that exist today, to help you get the idea:
+
+- A unique digital artwork.
+- A unique shirt in a limited-run fashion line.
+- An in-game item.
+- [An essay](https://zora.co/0x517bab7661C315C63C6465EEd1b4248e6f7FE183/145).
+- [A digital collectible](https://www.larvalabs.com/cryptopunks/details/1).
+- [A domain name](https://app.ens.domains/name/ethereum.eth)
+- A ticket that gives you access to an event or a coupon – these items are often known as semi-fungible. They are fungible until they are redeemed, at which point they can't be traded for a non-redeemed version.
+
+### ethereum.org examples {#ethereum-org-examples}
+
+We've used NFTs to give back to our contributors and store our website outside of centralized servers.
+
+#### POAPs (Proof of attendance protocol) {#poaps}
+
+If you contribute to ethereum.org, you can claim a POAP NFT. These are collectibles that prove you participated in an event. Some crypto meetups have used POAPs as a form of ticket to their events. [More on contributing](/contributing/#poap).
+
+![ethereum.org POAP](../../assets/use-cases/poap.png)
+
+#### ethereum.eth {#ethereum-dot-eth}
+
+This website is accessible via the ethereum.eth ENS (Ethereum name service) name. With the content hosted on IPFS, a decentralized storage solution, this website is actually an NFT.
```

Comment:
+ more cool things you can do with an ENS name:

> You can use your ENS name to receive any cryptocurrency, token, or blockchain-based item (like NFTs), point to a decentralized website (how to surf the decentralized web, e.g. almonit.eth.link), or store profile information like an avatar, email address, or Twitter handle. Since it can store any arbitrary information, ultimately any person or project can have their own ENS record.

https://medium.com/the-ethereum-name-service/reminder-that-ens-names-are-nfts-what-this-means-for-your-names-b7bcbea8715e#:~:text=ETH%20names%20on%20ENS%20are,So%20are%20DNS%20.&text=This%20means%20they%20work%20in,In%20fact%2C%20.